lib/membrane_rtc_engine/engine.ex

```diff
@@ -134,8 +132,8 @@ defmodule Membrane.RTC.Engine do
   end
   ```
 
-  Then, when we receive the first Media Event we can pass it to the RTC Engine
-  int the following way:
+  Then, when we receive the first Media Event we can pass it to the process maitaining
+  RTC Engine instance in the following way:
 
   ```elixir
   @impl true
```

Comment:
Still unsure about this example :P It still doesn't even use the engine's API. I think we have to either describe how such architecture could look like, so the reader knows what the 'process maintaining RTC Engine instance' is, or remove this and assume we have this covered in a demo/tutorial.

Reply:
Ok, let's leave this in `membrane_demo`. It's too much writing to describe this properly and everyone will take a look at our simple videoroom demo